src/systemTest/java/com/flightstats/hub/system/extension/DependencyInjectionExtension.java

```diff
@@ -0,0 +1,23 @@
+package com.flightstats.hub.system.extension;
+
+import com.google.inject.Injector;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.util.Optional;
+
+@Slf4j
+public class DependencyInjectionExtension implements BeforeAllCallback {
+    private static final ExtensionContext.Namespace NAMESPACE = ExtensionContext.Namespace.GLOBAL;
```

Comment:
I'm not sure I see the value in a static final to reference an enum. Why not just reference GLOBAL at the right point in the code?

Reply:
good point, this started as constant because it was to be a placeholder for future custom namespaces, but I'm not sure that is a road we will be able to go down and needs to get cleaned up.